packages/sdk/CHANGELOG.md

```diff
@@ -7,11 +7,13 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - `consumeProtectedData()`: Add two new status to ConsumeProtectedDataStatuses: "FETCH_WORKERPOOL_ORDERBOOK" and "PUSH_ENCRYPTION_KEY"
+- `processProtectedData()`: You can now pass it an `onStatusUpdate` callback to get similar "events" at each step of the process.
```

Comment:
I don't get the word "similar".
Oh because it refers to the previous sentence?

I would write:
```suggestion
- `processProtectedData()`: You can now pass an `onStatusUpdate` callback to get update events at each step of the process.
```

Reply:
Commit: f8b18749ea9a30e6541c90806035c94fb59da9e7